packages/field/src/FormRegistrarMixin.js

```diff
@@ -89,7 +93,14 @@ export const FormRegistrarMixin = dedupeMixin(
           return;
         }
         ev.stopPropagation();
-        this.addFormElement(child);
+
+        // Check for siblings to determine the right order to insert into formElements
+        // If there is no next sibling, index is -1
+        let indexToInsertAt = -1;
+        if (this.formElements && this.formElements instanceof Array) {
```

Comment:
```suggestion
        if (this.formElements && Array.isArray(this.formElements)) {
```

instanceof may fail for arrays coming from an iframe

Reply:
Changed!

Reply:
nice 🤗